src/Psalm/Internal/LanguageServer/Server/TextDocument.php

```diff
@@ -230,6 +230,28 @@ public function hover(TextDocumentIdentifier $textDocument, Position $position):
             return new Success(null);
         }
 
+        // Pretty print arrays (only).
+        if ( substr( $symbol_information['type'], 0, 6 ) === 'array{' ) {
+            $union = \Psalm\Type::parseString( $symbol_information['type'] );
```

Comment:
We're currently removing FQCN from the code and replacing them with `use`. Can you do the same here?